src/table/__integ__/performance-marks.test.ts

```diff
@@ -13,15 +13,19 @@ function setupTest(
   }) => Promise<void>
 ) {
   return useBrowser(async browser => {
-    if (process.env.REACT_VERSION === '18') {
-      return;
-    }
     const page = new BasePageObject(browser);
     await browser.url(`#/light/table/performance-marks${!inViewport ? '?outsideOfViewport=true' : ''}`);
     const getMarks = async () => {
-      await new Promise(r => setTimeout(r, 200));
-      const marks = await browser.execute(() => performance.getEntriesByType('mark') as PerformanceMark[]);
-      return marks.filter(m => m.detail?.source === 'awsui');
+      let awsuiMarks: PerformanceMark[] = [];
+      await page.waitForAssertion(async () => {
+        await new Promise(r => setTimeout(r, 200));
```

Comment:
This timeout is likely causing issues. Increasing it three-fold reduces flakiness, but the waitForAssertion seems to do a better job.